<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labb">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>File:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stabtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="fileLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>691</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="userBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>✔</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>741</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>370</y>
      <width>731</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>50</number>
     </property>
     <property name="verticalSpacing">
      <number>7</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="openbtn">
       <property name="text">
        <string>打开</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="fontcol">
       <property name="text">
        <string>字体颜色</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="infobtn">
       <property name="text">
        <string>文件信息</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="cosebtn">
       <property name="text">
        <string>关闭文件</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="savebtn">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="fontbtn">
       <property name="text">
        <string>字体</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="backgcbtn">
       <property name="text">
        <string>背景颜色</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="exitbtn">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="bgLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>bgLabel</zorder>
   <zorder>labb</zorder>
   <zorder>stabtn</zorder>
   <zorder>fileLabel</zorder>
   <zorder>userBtn</zorder>
   <zorder>textEdit</zorder>
   <zorder>layoutWidget</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
